pkg/webui/console/containers/device-importer/index.js

```diff
@@ -185,24 +262,43 @@ export default class DeviceImporter extends Component {
           field_mask.paths.push('network_server_address')
         }
       }
+    } catch (error) {
+      this.handleFatalError(error)
+    }
 
-      // Start batch device creation.
-      this.setState({
-        step: 'creation',
-        totalDevices: devices.length,
-      })
-      this.appendToLog('Creating end devices…')
-      const createStream = api.device.bulkCreate(appId, devices)
+    // Start batch device creation.
+    this.setState({
+      step: 'creation',
+    })
+    this.appendToLog('Creating end devices…')
+
+    try {
+      this.createStream = api.device.bulkCreate(appId, devices)
 
       await new Promise((resolve, reject) => {
```

Comment:
```suggestion
      await new Promise((resolve) => {
```

`reject` is not used